packages/util-dev/src/stringifyReplacers.ts

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const stringifyReplacer = (_key: any, value: any) => {
```

Comment:
This is not currently used, but I added to the ulti-dev packkage, because is a pice of code I find myself often rewriting when debugin transaction objects, this is a replacer funciton top be use with the JSON.stringify:

`console.log(JSON.stringify(x, stringifyReplacer))`

If the serializable object can be used in the same way I will remove this.

Reply:
Yep it can be used the same way. You can `JSON.stringify(toSerializableObj(x))`

Reply:
commit that introduced this change was removed